simpletransformers/classification/classification_model.py

```diff
@@ -996,6 +996,8 @@ def evaluate(
 
             if not multi_label:
                 preds = np.argmax(preds, axis=1)
+            else:
+                preds = (preds > 0.5).astype(int)
```

Comment:
I think we should just use the logic from the `predict()` function (not sure why I didn't do this in the first place.

```suggestion
            if multi_label:
                if isinstance(args.threshold, list):
                    threshold_values = args.threshold
                    preds = [
                        [self._threshold(pred, threshold_values[i]) for i, pred in enumerate(example)]
                        for example in preds
                    ]
                else:
                    preds = [[self._threshold(pred, args.threshold) for pred in example] for example in preds]
            else:
                preds = np.argmax(preds, axis=1)
```

This would actually use the `threshold` value from model_args.